website/src/blogs/openebs-0-5-enables-storage-policies-for-kubernetes-persistent-volumes.md

```diff
@@ -0,0 +1,74 @@
+
+Some notable changes include:
+
+- Storage Policy Enforcement Framework that allows DevOps teams to deploy a customized storage
+- Extend OpenEBS API Server to expose volume snapshot api
+- Support for deploying OpenEBS via helm charts
+- Support for monitor and get insights into OpenEBS volumes via Prometheus and Grafana
+- Sample Deployment YAMLs and corresponding Policy enabled Storage Classes for several stateful applications
+- Sample Deployment YAMLs for launching Kubernetes Dashboard for a preview of the changes done by OpenEBS Team to Kubernetes Dashboard
+
+***My favorite capability is the Storage Policy Framework that will enable each DevOps team to have their own storage controller — with their own storage policies. And the possibilities it will open-up!***
+
+Imagine as a Developer I want to test my service against MySQL database with different datasets. On my staging PV (mysqldata), I can create multiple snapshots containing different datasets — say snaps like *sandy* and *wendy*. Now I can extend OpenEBS to support a new policy “o*penebs.io/jiva-replica-snap”,* that can launch a new volume using snapshot for seed data.
+
+The policy can be defined for snap — *sandy* as follows:
+
+    # Define a SC referring to snapshot sandy 
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+      name: mysvc-mysqldata-kiran-ds-sandy
+    provisioner: openebs.io/provisioner-iscsi
+    parameters:
+      openebs.io/jiva-replica-count: "1"
+      openebs.io/jiva-source-pv: "mysqldata"      
+      openebs.io/jiva-replica-snap: "mysqldata-ds-sandy"
+
+and another one for *wendy*:
+
+    # Define a storage classes supported by OpenEBS
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+      name: mysvc-mysqldata-kiran-ds-wendy
+    provisioner: openebs.io/provisioner-iscsi
+    parameters:
+      openebs.io/jiva-replica-count: "1"
+      openebs.io/jiva-source-pv: "mysqldata"
+      openebs.io/jiva-replica-snap: "mysqldata-ds-wendy"
+
+In my PVC, I can now use the above StorageClasses(augmented with OpenEBS Storage Policies) to point to different dataset and independently test my service.
+
+All this from ***kubectl***.
+
+In the background, OpenEBS will create a separate Containerized Storage Engine/Controller for my test database — by sourcing the data from the specified snapshot.
+
+And btw, we are on the [CNCF LandScape](https://github.com/cncf/landscape) under the Cloud Native Storage options, and decidedly leading the niche market for Containerized Storage Controller. You will hear more on this in the coming days at the KubeCon, Austin.
+
+![CNCF](https://cdn-images-1.medium.com/max/800/1*rdKFLGyf0hRDB_zcgGlywA.png)
+
+You will hear more on OpenEBS 0.5 in the coming days at `KubeCon 2017`, Austin, Texas.
+
+I will be at the OpenEBS booths with my team and look forward to catching up with some of you in person.
```

Comment:
```suggestion
I will be at the OpenEBS booth with my team and look forward to catching up with some of you in person.
```

Reply:
This is not required. We never know it was a booth or booths.